.github/workflows/unit_tests.yml

```
@@ -88,7 +88,7 @@ jobs:
     - name: Test with pytest
       run: python -m pytest tests/models --showlocals -s -vv
     - name: Archive artifacts
-      uses: actions/upload-artifact@v3
+      uses: @v4
```

actionlint (1.7.8)

could not parse as YAML: found character that cannot start any token

(syntax-check)